pcli/src/command/query/governance.rs

```diff
@@ -64,7 +63,7 @@ impl GovernanceCmd {
                     let mut unfinished = client
                         .prefix_value(PrefixValueRequest {
                             prefix: all_unfinished_proposals().into(),
-                            chain_id: app.view().chain_params().await?.chain_id,
+                            ..Default::default()
```

Comment:
cc @plaidfinch ; IMO we should keep a separation between `pcli q` and `pcli v` here